rootfs/Dockerfile

```diff
@@ -1,27 +1,35 @@
 FROM quay.io/deis/base:v0.3.4
 
-RUN apt-get update \
-	&& apt-get install -y \
-		gcc \
-		libffi6 \
-		libffi-dev \
-		libssl1.0.0 \
-		libssl-dev \
-		musl \
-		musl-dev \
-		python \
-		python-dev \
-		--no-install-recommends \
-	&& curl -sSL https://bootstrap.pypa.io/get-pip.py | python - pip==8.1.2 \
-	&& pip install --disable-pip-version-check --no-cache-dir docker-py==1.10.3 \
-	&& apt-get remove -y --auto-remove --purge \
-		gcc \
-		libffi-dev \
-		libssl-dev \
-		musl-dev \
-		python-dev \
-	&& apt-get clean \
-	&& rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/man /usr/share/doc
+RUN buildDeps='gcc git libffi-dev libssl-dev musl-dev python-dev python-pip python-wheel python-setuptools'; \
+    apt-get update && \
+    apt-get install -y --no-install-recommends \
+        $buildDeps \
+        libffi6 \
+        libssl1.0.0 \
+        musl \
```

Comment:
Does this actually use `musl`? I thought that would only be needed on alpine, and deis/base is all about ubuntu-slim now as you know.